### ISSUE TITLE
When I simultaneously log into the admin site from two tabs/devices original tab stucks in redirect loop

### ISSUE THREAD

**Issue by fdobrovolny** [globaleaks/GlobaLeaks#3180]
**Describe the bug**
When I'm logged in to one tab let's call it tab A and then log in to the second tab let's call it tab B. Then when I go to a different menu page in tab A the app logs out and goes into a redirect loop and is spamming the server with requests. It is also very hard to close the tab.

**To Reproduce**
Steps to reproduce the behaviour:
1. DEVICE A/TAB A: Log into the admin site.
2. DEVICE B/TAB B: Log into the admin site.
3. DEVICE A/TAB A: Click on one of the menu items.
4. DEVICE A/TAB A: observe the redirect loop.

**Expected behaviour**
One redirect to log in form.

**Desktop (please complete the following information):**
 - OS: Linux, Fedora
 - Browser: Chrome 98
 - Version: 4.7.11 - reproduced on ***.try.globaleaks.org today at circa 18:10 CEST

**Additional context**
Log from browser console.
```
/api/preferences:1 Failed to load resource: net::ERR_INSUFFICIENT_RESOURCES
/api/admin/users:1 Failed to load resource: net::ERR_INSUFFICIENT_RESOURCES
/api/admin/node:1 Failed to load resource: net::ERR_INSUFFICIENT_RESOURCES
... skipped, the same as above a lot ...
/api/preferences:1 Failed to load resource: the server responded with a status of 412 ()
/api/admin/users:1 Failed to load resource: the server responded with a status of 412 ()
/api/admin/node:1 Failed to load resource: the server responded with a status of 412 ()
/api/admin/users:1 Failed to load resource: net::ERR_INSUFFICIENT_RESOURCES
/api/preferences:1 Failed to load resource: net::ERR_INSUFFICIENT_RESOURCES
/api/admin/users:1 Failed to load resource: net::ERR_INSUFFICIENT_RESOURCES
/api/preferences:1 Failed to load resource: the server responded with a status of 412 ()
/api/admin/users:1 Failed to load resource: the server responded with a status of 412 ()
/api/admin/node:1 Failed to load resource: the server responded with a status of 412 ()
/api/preferences:1 Failed to load resource: the server responded with a status of 412 ()
scripts.min.js:64 Throttling navigation to prevent the browser from hanging. See https://crbug.com/1038223. Command line switch --disable-ipc-flooding-protection can be used to bypass the protection
e.loginRedirect @ scripts.min.js:64
responseError @ scripts.min.js:64
(anonymous) @ scripts.min.js:8
(anonymous) @ scripts.min.js:8
$digest @ scripts.min.js:8
$apply @ scripts.min.js:8
_ @ scripts.min.js:8
C @ scripts.min.js:8
b.onload @ scripts.min.js:8
/api/admin/users:1 Failed to load resource: the server responded with a status of 412 ()
/api/admin/node:1 Failed to load resource: the server responded with a status of 412 ()
/api/preferences:1 Failed to load resource: the server responded with a status of 412 ()
/api/admin/node:1 Failed to load resource: net::ERR_INSUFFICIENT_RESOURCES
/api/preferences:1 Failed to load resource: net::ERR_INSUFFICIENT_RESOURCES
/api/admin/users:1 Failed to load resource: net::ERR_INSUFFICIENT_RESOURCES
...skipped, same as 6 lines above ...
scripts.min.js:64 Throttling navigation to prevent the browser from hanging. See https://crbug.com/1038223. Command line switch --disable-ipc-flooding-protection can be used to bypass the protection
e.loginRedirect @ scripts.min.js:64
/api/admin/users:1 Failed to load resource: the server responded with a status of 412 ()
/api/admin/node:1 Failed to load resource: the server responded with a status of 412 ()
/api/preferences:1 Failed to load resource: the server responded with a status of 412 ()
```

**Comment by evilaliv3** [comment]
I've just fixed the issue with commit https://github.com/globaleaks/GlobaLeaks/commit/333261a120c67c1bdff26e7c9085eede175b489f

The patch will be included by today in release 4.7.12

**Comment by fdobrovolny** [comment]
Thanks.
